06 LEAN Engine/01 Getting Started/metadata.json

```diff
@@ -0,0 +1,12 @@
+{
+    "type": "metadata",
+    "values": {
+        "description": "Lean Engine is an open-source algorithmic trading engine built for easy strategy research, backtesting and live trading.",
```

Comment:
Prepend "The" to the start of the sentence.

The docs use the oxford comma pattern "backtesting**,** and live trading".